frontend/components/Registration.js

```diff
@@ -0,0 +1,102 @@
+{/*Need to combine with Auth0 functionality for email and password confirmation*/}
+
+import React, { Component } from 'react'
+import Link from 'next/link';
+
+export default class Login extends React.Component {
+  constructor(props) {
+    super(props)
+
+    this.state = {
+      email: "",
+      password: "",
+      confirmPassword: "",
+      confirmationCode: "",
+      newUser: null
+    }
+    this.handleChange = this.handleChange.bind(this);
+    this.handleChange = this.handleChange.bind(this);
+  };
+
+  validateForm() {
+    return (this.state.email.length > 0 && this.state.password.length > 0 && this.state.password == this.state.confirmPassword); // ensures that the correct input is in the section, modify to add autho0 verification
+  }
+
+  validateConfirmationForm() {
+    return this.state.confirmationCode.length > 0; 
+  }
+
+  handleChange(event) {
+    this.setState({[ event.target.name]: event.target.value });
+  }
+
+  handleSubmit(event) {
+    event.preventDefault();
+    this.setState({ isLoading: true });
+    this.setState({ newUser: "test" });
+    this.setState({ isLoading: false });
+  }
+
+  handleConfirmationSubmit(event) {
+    event.preventDefault();
+    this.setState({ isLoading: true });
+  }
+
+  renderConfirmationForm() {
+    return (
+      <form onSubmit = { this.handleConfirmationSubmit }>
+        <div controlID = "confirmationCode">
+          <ControlLabel>Confirmation Code</ControlLabel>
+        </div>
+            {/* type = "tel"
```

Comment:
Try to avoid commented out code because Ive seen it a lot of times it stays in there foreever

Reply:
Okay I'll remove it all